package.json

```diff
@@ -35,7 +35,7 @@
 		"package-setup": "node -e \"require('./build/scripts/package.ts').setup()\"",
 		"package-create": "node -e \"require('./build/scripts/package.ts').create()\"",
 		"pretest": "yarn compile && yarn webpack",
-		"test": "node ./out/tests/runTest.js",
+		"test": "node ./out/tests/runTest.js --disable_cert_verification",
```

Comment:
As I rethink this, I'm a bit more concerned about disabling certificate validation. If a cert validation fails, it is a potential sign that you are under a MITM attack (which is sort of what's going on with your VPN, you just trust your man in the middle). I think it's fine to add this option, but I'm not sure it should be the default. Maybe just document this option somewhere?

Reply:
I pondered on this decision quite a bit, but finally decided to allow it as surgically as I can here.  My reason for allowing it is because it affects most of my development team, which are the main consumers of the test at this point.

Reply:
Could this be made an environment variable too? Although most current VS Code extension devs run in to this, it feels dangerous to disable security by default. Someone should have to make an active choice to do that.

Also, as you mentioned in another comment, maybe version pinning makes this less of an issue since versions are cached and the pinned version won't be updated nearly as often as a stable would?

Reply:
Funny you mention this, but it already _is_ an environment variable.  See lines 40-42 in runTest.ts.  Setting an environment variable is the mechanism used to communicate to node not to do cert verifications.  I'll change this back to doing cert verifications, then show my team how to do a surgical work-around because I don't want them setting `NODE_TLS_REJECT_UNAUTHORIZED=0` globally (e.g., via .bashrc or something) as that will affect _all_ node HTTPS connections.

Reply:
Certs are now being checked again by default.  If you're having problems with cert verifications, due to endpoint protection, there are ways to have node trust your endpoint protection certificate, or you can disable the cert verifications by either running the test script via `node ./out/tests/runTest.js --disable_cert_verification`, or via `NODE_TLS_REJECT_UNAUTHORIZED=0 npm run test`.  **Do not export NODE_TLS_REJECT_UNAUTHORIZED=0 into your environment as it will affect all node HTTPS connections done in that shell session.**